test/unit/client.test.ts

```diff
@@ -851,6 +850,91 @@ describe('Client', () => {
         conversationId: 'test-convo-id-123'
       })
     })
+
+    it('will update the SDK defaults if the actualResolution differs from the requested', async () => {
+      sdk = constructSdk();
+      const stream1 = new MockStream() as any as MediaStream;
+      stream1.getVideoTracks = jest.fn().mockReturnValue([{
+        applyConstraints: jest.fn(),
+        getConstraints: jest.fn(),
+        getSettings: jest.fn().mockReturnValue({
+          width: 1920,
+          height: 1080
+        })
+      }]);
+
+      const stream2 = new MockStream() as any as MediaStream;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9764801)
```suggestion
      const stream2 = new MockStream() as unknown as MediaStream;
```